test.js

```diff
@@ -2,21 +2,23 @@ import {serial as test} from 'ava';
 import got from 'got';
 import micro from 'micro';
 import testListen from 'test-listen';
-import m from '.';
+import access from '.';
```

Comment:
I usually use `m` (as in module) in my tests for consistency :). Could you change it back?